### PR TITLE
Add recipe for fmrib-unpack-fmrib-config

### PR DESCRIPTION
**Pull request by pauldmccarthy**
Howdy, the `fmrib-unpack-fmrib-config` package contains configuration files for [`fmrib-unpack`](https://github.com/conda-forge/fmrib-unpack-feedstock/), which is a data processing tool for the [UK BioBank](https://biobank.ctsu.ox.ac.uk/). The `fmrib-unpack-fmrib-config` package is already up on [PyPi](https://pypi.org/project/fmrib-unpack-fmrib-config/). Thanks!

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by pauldmccarthy** [comment]
@conda-forge/staged-recipes, this recipe is ready for review - thanks!

File: recipes/fmrib-unpack-fmrib-config/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,41 @@
+{% set name = "fmrib-unpack-fmrib-config" %}
+{% set version = "1.1.1" %}
+{% set sha256 = "a6665b6687d2f71de9becff0a4e243af23fe98e78a251499346a1b499a8b4758" %}
```
Comment (ocefpaf) [inline recipes/fmrib-unpack-fmrib-config/meta.yaml:3]:
```suggestion
```

File: recipes/fmrib-unpack-fmrib-config/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
```
Comment (ocefpaf) [inline recipes/fmrib-unpack-fmrib-config/meta.yaml:11]:
```suggestion
  sha256: a6665b6687d2f71de9becff0a4e243af23fe98e78a251499346a1b499a8b4758
```

**Review by ocefpaf — CHANGES_REQUESTED** [review]
No need for a jinja variable fot something that is used once. It decreases recipe readability without adding any advantage.

**Pull request merged**